File: Divundo/app/src/main/java/com/example/sergio/divundo/activities/MainActivity.java
package com.example.sergio.divundo.activities;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.support.v7.app.AppCompatActivity;

import com.example.sergio.divundo.R;
import com.example.sergio.divundo.fragments.LogInFragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Fragment fragment = new LogInFragment();
        getSupportFragmentManager().beginTransaction()
                .replace(R.id.fragment_container, fragment).commit();
    }

}

File: Android_Kotlin/DivundoKotlin/app/src/main/java/com/example/sergio/divundokotlin/activities/MainActivity.kt
package com.example.sergio.divundokotlin.activities

import android.content.Context
import android.content.SharedPreferences
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v4.app.Fragment
import com.example.sergio.divundokotlin.R
import com.example.sergio.divundokotlin.fragments.EntrySuccessFragment
import com.example.sergio.divundokotlin.fragments.LoginFragment

class MainActivity : AppCompatActivity() {

    private lateinit var prefs: SharedPreferences
    private var token: String = ""

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        //shared preferences
        val fileName = "mis-preferencias"
        prefs = getSharedPreferences(fileName, Context.MODE_PRIVATE)
        token = prefs.getString("token", "")

        if (!token.isEmpty()) {
            fragmentTransaction(EntrySuccessFragment())
        } else {
            fragmentTransaction(LoginFragment())
        }

    }

    private fun fragmentTransaction(fragment: Fragment) {
        supportFragmentManager.beginTransaction()
            .replace(R.id.container, fragment)
            .commit()
    }
}
